src/shared/components/search-result/search-result.component.html

```diff
@@ -1,6 +1,15 @@
 <table *ngIf="hasResults() || hasDrafts()">
   <caption>
     <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseState ? 'Your cases' : 'Search result' }}</h2>
+
+    <div class="govuk-warning-text" *ngIf="paginationLimitEnforced">
```

Comment:
unit test could be added to check the warning message renders as expected if `paginationLimitEnforced` is true